manuscript/chapter2.md

```diff
@@ -10,7 +10,7 @@ I>     1. A grammar, which is a rule that tells us how we can construct formulas
 I> 1. A set of **axioms**, that is, a starting set that we take for granted without having any proofs for
 I> 1. A set of **inference rules**
 
-After a formal system is defined, other formal systems can extend it. For example, the ZFC set theory is based on first-order logic, which is based on propositional logic which represents a formal system. We'll discuss this theory briefly in the next chapter.
+After a formal system is defined, other formal systems can extend it. For example, set theory is based on first-order logic, which is based on propositional logic which represents a formal system. We'll discuss this theory briefly in the next chapter.
```

Comment:
@Chobbes do you think it makes sense to keep Peano/Set theory intro here? In the section below (right after Definition 5) we refer to `sets`, and we also mention `Peano` (Definition 7).

Reply:
Hmmm. Not sure. I think it could use some reorganization to stay clearer. Maybe try moving Peano down to Definition 7...?

I'm not sure how I feel about the higher-order logic section. It seems like it's just copy-pasted from Wikipedia, and I'm not sure if it really gives the reader much intuition. Hmmm. I'll have to read it in more detail.